### ISSUE TITLE
Resource grid: Use "Logs" rather than "View" in the logs column

### ISSUE THREAD

**Issue by drewnoakes** [dotnet/aspire#2148]
A few ideas for improvements to this part of the table:

![image](https://github.com/dotnet/aspire/assets/350947/4693c26b-7c3b-4583-87e1-d3c18a87ba95)

The "Logs" and "Details" columns both contain "View" buttons. If logs column buttons read "Logs", it'd be easier to know which button to click when scanning left-to-right from the resource name at the far side of the table.

As it stands, you have to scan left-to-right, then up the table, then back down. If you have many resources, it can be a long way to the top, and you can lose track of which row you were on, so scan back left again to validate.

![image](https://github.com/dotnet/aspire/assets/350947/c13795e0-8fd0-40cd-a019-6a0c834463e0)

**Comment by kvenkatrajan** [comment]
@leslierichardson95  thoughts on verbiage here?

**Comment by leslierichardson95** [comment]
I agree the name should be changed from "View" to something else, but I feel like just "Logs" doesn't differentiate from the Logs header all that much (I'd suggest "view logs" but that would take up more space). That said, we can go with "Logs" until one of us comes up with something better. :)

**Comment by davidfowl** [comment]
Assigning to you @leslierichardson95

**Comment by kvenkatrajan** [comment]
@leslierichardson95  moving this to preview 6 - please confirm verbiage.

**Comment by leslierichardson95** [comment]
For now, let's replace "View" with "logs" and "Details" and make sure the a11y tooltips give specific details about what resource that particular link is corresponding to.

**Comment by davidfowl** [comment]
Is this still relevant now that we have https://github.com/dotnet/aspire/pull/5516?

**Comment by drewnoakes** [comment]
We no longer need this.
